cumulusci/tests/util.py

```diff
@@ -213,3 +226,25 @@ def _init_task():
         lambda: is_person_accounts_enabled,
     ), mock.patch.object(task, "_init_task", _init_task):
         yield
+
+
+@contextmanager
+def mock_homedir(home, cumulusci_key="0123456789ABCDEF"):
+    real_homedir = str(Path.home())
+    with mock.patch("pathlib.Path.home", lambda: Path(home)), mock.patch.dict(
+        os.environ,
+        {
+            "HOME": home,
+            "USERPROFILE": home,
+            "CUMULUSCI_KEY": cumulusci_key,
+            "REAL_HOME": real_homedir,
+            "REAL_CUMULUSCI_KEY": os.environ.get("CUMULUSCI_KEY", ""),
+        },
+    ):
+        yield
```

Comment:
Should this be called mock_env? It's not obvious from the name that it would touch CUMULUSCI_KEY

Reply:
135d4ebfc1dd38ad60537b113d0c647ffe48b2cf